app/views/foreman_salt/api/v2/salt_variables/show.json.rabl

```diff
@@ -10,7 +10,7 @@ attributes :id, :variable, :salt_state, :salt_state_id, :description, :override,
 node do |salt_variable|
   {
     :override_values => partial(
-      'api/v2/override_values/index',
+      'api/v2/override_values/main',
```

Comment:
The `index` file for override values was moved to `[foreman_puppet](https://github.com/theforeman/foreman_puppet/blob/master/app/views/foreman_puppet/api/v2/override_values/index.json.rabl)`, but we can use `main` instead, as index only inherits from `main`, anyways.